modules/vue-i18n/vite.config.js

```diff
@@ -4,5 +4,5 @@ import { defineConfig } from 'vite'
 import vueI18n from '@intlify/vite-plugin-vue-i18n'
 
 export default defineConfig({
-  plugins: [vueI18n()],
+  plugins: [vueI18n({ runtimeOnly: false })],
```

Comment:
This is not a proper fix. Revert this.

Proper fix is to remove parametrized translation keys (`t('You have {0} items in cart.', [cart.items.length])`). Instead use synthehic keys like in `modules/theme-tailwind/atoms/GiftPanel.vue`